app_backend/models/user.py

```diff
@@ -0,0 +1,34 @@
+from bson import ObjectId
+from pydantic import BaseModel, Field
+from typing import Optional
+
+
+class PyObjectId(ObjectId):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+    @classmethod
+    def validate(cls, v):
+        if not ObjectId.is_valid(v):
+            raise ValueError("Invalid objectid")
+        return ObjectId(v)
+    @classmethod
+    def __modify_schema__(cls, field_schema):
+        field_schema.update(type="string")
+
+
+"""
+    UserLogin model keeps track of the user login information
+    username: str
+        The username of the user
+    email: str
+        The email of the user
+    password: str
+        The password of the user
+"""
+class User(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    name: str
+    email: str
+    password: str
+    devices: Optional[list]
```

Comment:
Not implemented yet, but nice to have so we can get this merged ASAP and help other PR's is Dooley $ and meal swipes

Reply:
Also friends (list of uuids), and liked foods (list of strings?), station (string), is_sharing_location (boolean)

Reply:
OK will add these